doc/Core.xml

```diff
@@ -6242,6 +6242,42 @@ onvif://www.onvif.org/name/ARV-453
           </variablelist>
         </para>
       </section>
+      <section>
+        <title>Configuration Renewal</title>
+        <para>
+          The configuration allows for a renewal endpoint to be set. If the device supports this feature, it shall automatically renew the credentials 
+          when they are about to expire.
+        </para>
+        <para>
+          The device shall do a GET request to the configured <literal>RenewalEndpoint</literal> with the access token retrieved from the configured 
+          <literal>AuthorizationServer</literal>. The endpoint shall respond with a JSON payload with the following structure:
+          <programlisting><![CDATA[{
+    "region": "string" | null,
+    "storageUri": "string" | null,
+    "user": {
+        "username": "string" | null,
+        "password": "string" | null,
+        "token": "string" | null
+    },
+    "expiresAt": "<ISO 8601 date-time>"
+}]]></programlisting>
+          Any null value in the response shall clear the corresponding optional parameter in the <literal>StorageConfiguration</literal>.
```

Comment:
Here are the cases that I see ambiguity 

- Renewal Response leaves out a field from payload
- RenewalResponse includes a field with a null string/value

Both are treated the same? 

In either case, for the same argument as in https://github.com/onvif/specs/pull/423/files#r2077654596, response from renewal should not modify StorageConfiguration but the 'local/temporary/non-persistent' data that device maintains.

Any updates to StorageConfiguration should only happen via SetStorageConfiguration (e.g. over Uplink) to avoid any race conditions (or nondeterministic) for updating same data from 2 different paths.

```suggestion
          Any null value in the response shall clear the corresponding optional parameter stored in the device.</literal>.
```